scripts/zkai_d64_rmsnorm_to_projection_bridge_input.py

```
@@ -0,0 +1,319 @@
+        raise BridgeInputError("source schema mismatch")
+    if source.get("target_id") != TARGET_ID:
+        raise BridgeInputError("source target mismatch")
+    if source.get("width") != WIDTH or source.get("row_count") != WIDTH:
+        raise BridgeInputError("source width/row_count mismatch")
+    rows = source.get("rows")
+    if not isinstance(rows, list) or len(rows) != WIDTH:
+        raise BridgeInputError("source rows mismatch")
+    values = []
+    for index, row in enumerate(rows):
+        if not isinstance(row, dict) or row.get("index") != index:
+            raise BridgeInputError("source row index mismatch")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)